README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
```

Comment:
It is always better to provide some high level details about your project(in this case TODO) in README file if you are implementing something from scratch, this is automatically generated README file but just a small suggestion.